src/backend.js

```diff
@@ -114,7 +115,8 @@ export async function uploadThumbnail(thumbnail, id) {
   uploadBytes(storageRef, thumbnailBlob);
 };
 
-export async function uploadStl(stlFile, id) {
-  var storageRef = ref(firebaseStorage, `print-files/${id}.stl`);
-  uploadBytes(storageRef, stlFile);
+export async function uploadStl(file, jobName, id) {
+  console.log(jobName + "_" + id + ".stl")
+  var storageRef = ref(firebaseStorage, `print-files/${jobName + "_" + id}.stl`);
+  uploadBytes(storageRef, file);
```

Comment:
Does adding `jobName` to the stored filename provide any benefit? Id is already a unique identifier.

Reply:
Hi, yes the jobName is there to provide a readable file name for when the user downloads it to their local machine. That way the user knows which file is associated with which job they've accepted

Reply:
Okay I'll switch it back then approve the PR

Reply:
Thanks Matt, I appreciate the review